lldb/tools/lldb-dap/src-ts/webview/symbols-table-view.ts

```diff
@@ -0,0 +1,90 @@
+import type { CellComponent, ColumnDefinition } from "tabulator-tables";
+import type { DAPSymbolType } from ".."
+
+function get_tabulator_hexa_formatter(padding: number): (cell: CellComponent) => string {
+  return (cell: CellComponent) => {
+    const val = cell.getValue();
+    if (val === undefined || val === null) {
+      return "";
+    }
+
+    return val !== undefined ? "0x" + val.toString(16).toLowerCase().padStart(padding, "0") : "";
+  };
+}
+
+const SYMBOL_TABLE_COLUMNS: ColumnDefinition[] = [
+  { title: "User ID", field: "userId", sorter: "number", widthGrow: 0.8 },
+  {
+    title: "Name",
+    field: "name",
+    sorter: "string",
+    widthGrow: 3,
+    minWidth: 200,
+    tooltip : (_event: MouseEvent, cell: CellComponent) => {
+      const rowData = cell.getRow().getData();
+      return rowData.name;
+    }
+  },
+  {
+    title: "DSX",
```

Comment:
I feel like DSX isn't a very obvious name, I wasn't sure what this column was in the video reference.

Maybe we could have a slightly more descriptive title? Or call this like `Symbol Attributes` or something? Not sure how to keep this compact while still being descriptive.

Reply:
Just separated it to 3 columns 
<img width="1739" height="1045" alt="Screenshot From 2025-08-17 01-17-54" src="https://github.com/user-attachments/assets/9eeb2a4a-0172-45ec-b367-5aa9ec97cc98" />